data/json/items/comestibles/drink.json

```diff
@@ -1042,6 +1042,30 @@
     "relative": { "fun": 1 },
     "use_action": [  ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_green",
+    "name": "green tea",
+    "copy-from": "tea",
+    "color": "green",
+    "fatigue_mod": 3,
+    "stim": 1,
+    "calories": 2,
+    "description": "Made from applying hot water to leaves of the tea plant /Camellia sinensis/, but with a different taste to his brother black tea.",
+    "fun": 6
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_fruit",
+    "name": "fruit tea",
+    "copy-from": "tea",
+    "color": "red",
+    "calories": 6,
+    "fatigue_mod": 0,
```

Comment:
These are probably going to produce a "does not update value" error. You don't need to specify an attribute of zero.

Reply:
even when the original (tea) has a fatigue mod itself? wouldn't it take the tea value if not added?

Reply:
Ah, forgot it was a copy-from, you're probably fine